### PR TITLE
Enable running of lanes from project fastfiles

### PR DESCRIPTION
**Pull request by taquitos**
- Can now run lanes from the project.
- Entire flow for triggering builds with your chosen lane works 🎉
- includes fix for #225

### CODE REVIEW

File: features/build_runner/fastlane_build_runner.rb [hunk 1]
```diff
@@ -70,7 +70,7 @@ def run
         logger.error(ex)
         logger.error(ex.backtrace)
         return [] # TODO: return artifacts here (if any)
-      ensure
+        # ensure
```
Comment (taquitos) [inline features/build_runner/fastlane_build_runner.rb]: rubocop++

File: features/project/project_controller.rb [hunk 2]
```diff
@@ -12,6 +12,8 @@ class ProjectController < AuthenticatedControllerBase
     # TODO: this should actually be a POST request
     get "#{HOME}/:project_id/trigger" do
       project_id = params[:project_id]
+      lane = params[:lane] # TODO: not sent yet
+      platform = params[:platform] # TODO: not sent yet
```
Comment (taquitos) [inline features/project/project_controller.rb]: right now we get `lane` from the project, but we don't store `platform` maybe we need to add this to the `Project` model

File: features/project/project_controller.rb [hunk 3]
```diff
@@ -29,9 +31,11 @@ class ProjectController < AuthenticatedControllerBase
       build_runner = FastlaneBuildRunner.new(
         project: project,
         sha: current_sha,
-        github_service: current_github_provider_credential
+        github_service: FastlaneCI::GitHubService.new(provider_credential: current_github_provider_credential)
```
Comment (taquitos) [inline features/project/project_controller.rb]: Hah.

File: features/project/project_controller.rb [hunk 4]
```diff
@@ -46,36 +50,23 @@ class ProjectController < AuthenticatedControllerBase
       # Long term, the best appraoch would probably to have the FastfileParser be
       # its own Ruby gem, or even part of the fastlane/fastlane main repo
       # For now, this is good enough, as we'll be moving so fast with this one
-      project_path = project.repo_config.local_repo_path
```
Comment (taquitos) [inline features/project/project_controller.rb]: moved this stuff in to `Project` model

File: features/project/project_controller.rb (hunk 3 above)
Comment (KrauseFx) [reply features/project/project_controller.rb]: types, am I right?

Comment (taquitos) [reply features/project/project_controller.rb]: I'm gonna build a typechecker for ruby. I'll call it `JavaCop`

**Pull request merged**